src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingOptInRequestsOasis.java

```diff
@@ -20,312 +20,312 @@
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsNordicsDataProvider",
+//      groups = "AdsOptInRejectedOasis"
+//  )
+//  public void adsTrackingRejectedForNordics(List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.rejectOptInModal(driver, DENMARK, ADS_HOME_PAGE);
+//
+//    modal.verifyTrackingRequestsNotSend(urlPatterns, networkTrafficInterceptor);
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsPrebidDataProvider",
+//      groups = "AdsOptInRejectedOasis"
+//  )
+//  public void adsTrackingRejectedForPrebid(String[] instantGlobals, List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.rejectOptInModal(driver, POLAND, ADS_HOME_PAGE, instantGlobals);
+//
+//    modal.verifyTrackingRequestsNotSend(urlPatterns, networkTrafficInterceptor);
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsA9DataProvider",
+//      groups = "AdsOptInRejectedOasis"
+//  )
+//  public void adsTrackingRejectedForA9(String[] instantGlobals, List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.rejectOptInModal(driver, POLAND, ADS_HOME_PAGE, instantGlobals);
+//
+//    modal.verifyTrackingRequestsNotSend(urlPatterns, networkTrafficInterceptor);
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsQuantcastDataProvider",
+//      groups = "AdsOptInRejectedOasis"
+//  )
+//  public void adsTrackingRejectedForQuantcast(List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.rejectOptInModal(driver, POLAND, ADS_HOME_PAGE);
+//
+//    modal.verifyTrackingRequestsNotSend(urlPatterns, networkTrafficInterceptor);
+//
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsComscoreDataProvider",
+//      groups = "AdsOptInRejectedOasis"
+//  )
+//  public void adsTrackingRejectedForComscore(List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.rejectOptInModal(driver, POLAND, ADS_HOME_PAGE);
+//
+//    modal.verifyTrackingRequestsNotSend(urlPatterns, networkTrafficInterceptor);
+//
+//  }
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)